src/test/java/org/broadinstitute/consent/http/resources/DatasetResourceTest.java

```diff
@@ -364,6 +365,14 @@ public void testValidateDatasetNameNotFound() {
         resource.validateDatasetName("test");
     }
 
+    @Test
+    public void testFindAllActiveStudyNamesSuccess() {
+        when(datasetService.findAllActiveStudyNames()).thenReturn(Set.of("Hi", "Hello"));
+        initResource();
+        Response response = resource.findAllActiveStudyNames();
+        assertEquals(200, response.getStatus());
+    }
```

Comment:
Might be nice to test an exception case here too. For example, we don't want to percolate Sql exceptions back up to the client since they often contain somewhat sensitive info useful in security-fishing. The base `Resource` class should help handle that for you.